### DISCUSSION TITLE
IFR/VFR setting from SimBrief FPL via CDU

### DISCUSSION THREAD

**Discussion by lve0200** [flybywiresim/aircraft#2646] [Help]
It appears to me that SimBrief FPL downloaded via MCDU are only recognised by MSFS as VFR, or what I am doing wrong here?

**Comment by CptBayer** [comment]
Hi Simmers,

I laso have problems with tat simbrief thing. I entered my username but I didn´t figgered out how a fligtplan will be downloaded via the fmcg ?!? Can someone give me a hint please .... When I enter a new Dep/Arr for exampel EVRA/EDDM (FP cretaed with simbrief), I got no flightplan sowed ?! 

Thanks Pepe

**Reply by lve0200** [reply]
On SimBrief you must create an OFP.
In the MCDU Options you must set your username for SimBrief.
Then press on the INIT page "Init request". It will take a few seconds and you should see "somethink like ...uplink... and your init page will be filled.

**Reply by DIEHARDLY** [reply]
I have a problem with this too. Have even created a new SIMBRIEF account and username. I create a new flight in Simbrief, Generate OFP, then in FS2020 go to Options>AOC>SIMBRIEF and enter my username (which I am sure is correct). Then back out of that and onto the INIT page but I get no "INIT REQUEST" option. I have seen 320SIMPILOT do this on YouTube and have followed every step that he does but his INIT screen is very different to mine (I hope I've attached screenshots of his INIT screen and mine)
![My INIT screen](https://user-images.githubusercontent.com/76540562/103045481-93087b80-457c-11eb-9f73-792ec615cbb4.png)
![320simpilot INIT screen](https://user-images.githubusercontent.com/76540562/103045484-97349900-457c-11eb-89c4-e6e8e44168e4.png)
Any ideas please ? I must be doing something stupid.

**Reply by CptBayer** [reply]
Thanx folks,

I will give it a dry and report .... 

Greets klaus

(My 3090 is on te way ... yeah ) ...

**Reply by philk209** [reply]
I'm afraid this is exactly the same thing with me....most frustrating!  Tried everything!
Phil

**Comment by CptBayer** [comment]
Hi folks,


for me it is working fine to get te simbrief FP imported. But I also have a uge fps drop with a loaded FP ... hmmm ...

Greets Klaus

**Comment by AdamNZ** [comment]
In the MSFS initial planning screen, are you *only* selecting the DEPARTURE airport? Don't select an ARRIVAL airport.

**Reply by tracernz** [reply]
If you do want to load again, put some different airports as departure/arrival, then the INIT REQUEST should become available again.

**Reply by philk209** [reply]
AdamNZ...thanks ever so much for your response. Your suggestion has worked! I owe you a drink! Thanks again. Phil

**Reply by AdamNZ** [reply]
Wow!!! I'm pretty dumb when compared to all the devs and contributors here but so glad that, occasionally, I get to help someone out!

**Comment by Inaxair** [comment]
Hello @philk209 @CptBayer 

I will recommend you watch our tutorial over the use of the Simbrief feature!

https://www.youtube.com/watch?v=l_5J7ZuEEtk

Best regards,
Inaxair from FlyByWire Simulations

**Reply by CptBayer** [reply]
Hi Inaxair,

Thanx for the link. I Got it and everything worked. Till today. When i press INIT REQUEST on the INIT page, I only see Departure and Arrival airport displayed. all the waypoints from the flightplan are missing?! Any Idea? I am using navigraph as well and in this app, the flightplan loads corrct from simbrief with all waypoints ..

Greet Klaus

**Comment by lve0200** [comment]
After a lot of sidetrack, I would like to bring my original question back! 
It appears to me that SimBrief OFP downloaded via MCDU are only recognised by MSFS-ATC as VFR, or what am I doing wrong here?
Where are these FPL stored, I guess temporary?
Or do I have to open an issue for this??
